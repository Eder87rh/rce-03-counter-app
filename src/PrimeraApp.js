import React, { Fragment } from "react";

const PrimeraApp = ({ saludo = "Hola Mundo" }) => {
  return (
    <Fragment>
      <h1>{saludo}</h1>
      <p>Mi primera aplicacion en react</p>
    </Fragment>
  );
};

export default PrimeraApp;
