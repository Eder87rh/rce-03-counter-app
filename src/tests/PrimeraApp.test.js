import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";

import PrimeraApp from "../PrimeraApp";

describe("Pruebas en PrimaraApp", () => {
  test('debe de mostrar el mensaje "Hola, Soy Goku"', () => {
    const saludo = "Hola, soy goku";
    const wrapper = shallow(<PrimeraApp saludo={saludo} />);
    expect(wrapper).toMatchSnapshot();
  });
});
