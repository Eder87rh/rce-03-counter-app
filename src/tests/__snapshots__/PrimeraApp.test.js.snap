// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en PrimaraApp debe de mostrar el mensaje "Hola, Soy Goku" 1`] = `
<Fragment>
  <h1>
    Hola, soy goku
  </h1>
  <p>
    soy un subtitulo
  </p>
</Fragment>
`;
